util/cmake/FindSeqAn.cmake

```diff
@@ -366,6 +372,16 @@ if (OPENMP_FOUND)
     endif ()
 endif ()
 
+if (Boost_FOUND)
+  # Example warning:
+  # C:\seqan-deps\boost_1_53_0\boost/mpl/if.hpp(131,1): error : pasting formed
+  # 'BOOST_PP_TUPLE_ELEM_E_2(', an invalid preprocessing token
+  if (COMPILER_CLANG AND STDLIB_VS)
+    message (STATUS "The boost library (at least until 1.53) doesn't support the clang/c2 compiler on windows (version 3.7 and 3.8), yet!")
+    set (Boost_FOUND FALSE)
```

Comment:
also unset the following, just in case someone includes boost as optional:
Boost_INCLUDE_DIRS
Boost_LIBRARY_DIRS
Boost_LIBRARIES